requirements.txt

```diff
@@ -0,0 +1,66 @@
+appdirs==1.4.3
```

Comment:
There can't be that much dependencies needed. Are you familiar with the concept of [virtual environment](https://docs.python.org/3/tutorial/venv.html)? It is super-useful